py_rql/exceptions.py

```diff
@@ -29,3 +29,11 @@ class RQLFilterLookupError(RQLFilterError):
 class RQLFilterValueError(RQLFilterError):
     """ Value error is raised when provided values can't be converted to DB field types. """
     MESSAGE = 'RQL Value error.'
+
+
+class RQLFilterNotFoundError(RQLFilterError):
```

Comment:
This error is doubtful for me as the original idea was to simply skip non described filters (this is how Django-RQL works, for example). I would suggest to keep following "open" approach and, thus, would suggest to remove this error type.